### ISSUE TITLE
Installing deb has error Dependency is not satisfiable: docker-ce-cli

### ISSUE THREAD

**Issue by bubblesmakesrecords** [docker/for-linux#1392]
* [x] This is a bug report
* [ ] This is a feature request
* [ ] I searched existing issues before opening this one

### Expected behavior

When I run the Docker for Linux .deb then software manager should allow me to install the package.

### Actual behavior

Software manager (DEB installer on Mint) has an error that says "Error: Dependency is not satisfiable: docker-ce-cli"

### Steps to reproduce the behavior

Linux Mint 20.3 Cinnamon
Latest docker DEB from https://docs.docker.com/desktop/linux/install/

**Comment by szluyu99** [comment]
+1

**Comment by Adrian-Bielefeldt** [comment]
Same problem and OS.

**Comment by cpuguy83** [comment]
Looks like the deb package depends on having the docker apt repos installed already.

**Comment by thaJeztah** [comment]
Yes, the installation instructions outline that the first step is to set up the package repository (to allow the dependencies to be installed); https://docs.docker.com/desktop/linux/install/ubuntu/#install-docker-desktop

<img width="477" alt="Screenshot 2022-05-17 at 22 23 55" src="https://user-images.githubusercontent.com/1804568/168903628-6a62b4d5-dafd-4a50-8fc8-abb34e7c7755.png">


FWIW, the plan of the Docker Desktop team is to make the deb and rpm packages available through the package repositories at download.docker.com, but some work is left to be done to integrate it in our release pipelines.

Let me move this ticket to the https://github.com/docker/desktop-linux/issues issue tracker, which I think is the right place for issues related to docker desktop for linux.

**Comment by thaJeztah** [comment]
oh, looks like I don't have permissions to that repository, so can't transfer it /cc @p1-0tr

**Comment by thaJeztah** [comment]
Alright, I should have permissions now; motoring this to the right issue tracker
